Patch: 65_handle_templates_directories.patch
Author: Tollef Fog Heen <tfheen@debian.org>
Skip directories when updating (Closes: #284771)
Index: mailman-2.1.12/bin/update
===================================================================
--- mailman-2.1.12.orig/bin/update	2009-03-02 20:50:13.000000000 +0100
+++ mailman-2.1.12/bin/update	2009-03-02 20:50:17.000000000 +0100
@@ -126,7 +126,7 @@
         try:
             fp = open(os.path.join(mm_cfg.TEMPLATE_DIR, gtemplate))
         except IOError, e:
-            if e.errno <> errno.ENOENT: raise
+            if not (e.errno in [errno.ENOENT, errno.EISDIR, ]): raise
             # No global template
             continue
 
